.storybook/stories/MyKivaBadgeCompleted.stories.js

```diff
@@ -0,0 +1,208 @@
+import BadgeCompleted from '#src/components/MyKiva/BadgeCompleted.vue';
+
+export default {
+	title: 'MyKiva/BadgeCompleted',
+	component: BadgeCompleted,
+};
+
+const story = (args = {}) => {
+	const template = (_args, { argTypes }) => ({
+		props: Object.keys(argTypes),
+		components: { BadgeCompleted },
+		setup() { return { args }; },
+		template: `
+			<badge-completed v-bind="args" />
+		`,
+	});
+	template.args = args;
+	return template;
+};
+
+const contentfulBadgeData = {
```

Comment:
This contentful example data could be moved to a mock data file to make this file easier to maintain (and we'll be able to reuse the data for other stories).